tests/integration/test_subscriptions_cli.py

```diff
@@ -250,3 +249,60 @@ def test_subscriptions_results_success(invoke, options, expected_count):
 
     assert result.exit_code == 0  # success.
     assert result.output.count('"id"') == expected_count
+
+
+def test_request_base_success(invoke, geom_geojson):
```

Comment:
May I also suggest a failure test for `request`?

Reply:
I didn't include a failure test for two reasons:
1. The success tests fully cover the code. 
2. Looking at the code, there isn't any logic other than calling a few functions that are fully tested. So, I'm not sure what failure I'd test for. Even testing for BadParameter would be basically testing the typing library.

Can you think of a beneficial test to include here?

Reply:
Ah yes, I realize that now. A success test seems to be the only thing that makes sense here 👍